x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.tsx

```diff
@@ -446,6 +446,7 @@ export const EditPackagePolicyForm = memo<{
         onChange={handleExtensionViewOnChange}
         validationResults={validationResults}
         isEditPage={true}
+        isAgentlessEnabled={true}
```

Comment:
not sure I understand why we hardcode `true` here. Shouldn't it be the same logic as in the create package policy flow - based on the combination of factors like FF and checks what type of env is it? basically what comes from `useAgentless` hook

Reply:
You're correct. I missed changing it to `isAgentlessEnabled`